src/guides/v2.4/release-notes/b2b-release-notes.md

```diff
@@ -67,8 +67,9 @@ These release notes can include:
 
 **Issue**: Magento sometimes displays a 404 error when a buyer creates a purchase order and then navigates to the checkout page. This error occurs when a buyer has previously created a different purchase order with an online payment method before navigating to the checkout page without completing the previous purchase. The buyer can still place the purchase order. **Workaround**: None. <!--- B2B-1605-->
 
-**Issue**: Discounts for a specific payment method persist during checkout for a purchase order even when the buyer changes payment method during final checkout. As a result, customers may receive a discount that they are not entitled to. This occurs because a cart rule for the original payment method is still applied despite the change in payment method. **Workaround**: None. <!-- B2B-1012 -->
+**Issue**: Discounts for a specific payment method persist during checkout for a purchase order even when the buyer changes payment method during final checkout. As a result, customers may receive a discount that they are not entitled to. This occurs because a cart rule for the original payment method is still applied despite the change in payment method. **Workaround**: None. See the [Magento 2.4.2 B2B known issue: discount remains pay method change](https://support.magento.com/hc/en-us/articles/360054667312) Knowledge Base article. <!-- B2B-1012 -->
```

Comment:
```suggestion
**Issue**: Discounts for a specific payment method persist during checkout for a purchase order even when the buyer changes their payment method during final checkout. As a result, customers may receive a discount that they are not entitled to. This occurs because a cart rule for the original payment method is still applied despite the change in payment method. **Workaround**: None. See the [Magento 2.4.2 B2B known issue: discount remains after payment method is changed](https://support.magento.com/hc/en-us/articles/360054667312) Knowledge Base article. <!-- B2B-1012 -->
```